private/pkg/transport/http/httpserver/httpserver.go

```diff
@@ -34,176 +37,64 @@ const (
 	DefaultIdleTimeout = 3 * time.Minute
 )
 
-// Mapper initializes a Router.
-type Mapper interface {
-	Map(chi.Router) error
+type runner struct {
+	shutdownTimeout   time.Duration
+	readHeaderTimeout time.Duration
+	tlsConfig         *tls.Config
+	maxBodySize       int64
```

Comment:
We could probably remove this, because it's a property of the middleware and less of the runner. I think it's trivial to find this instead of logging it.

Reply:
yeah, I missed that this was already in the new `router` package; makes sense that it's part of the middleware. I'll remove it.